ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -569,6 +572,15 @@ SSPC:
   implements:
   - OPERATIONAL
 
+SPC:
+  description: "Supply static pressure control via fan speed"
+  is_abstract: true
+  uses:
+  - static_pressure_sensor
```

Comment:
Is this different than supply air static pressure sensoring? 

Is it exhaust or supply air? I think we should be more specific here, since static pressure could be interpreted differently (have different operating ranges) if its exhaust versus supply. What does this device serve and where are the sensors located relative to the device?

Reply:
This is not different from supply air static pressure sensor. This is a used as this is modeled as a fan. 
This Fan is located at Terrace floor from there this device serve Outside Fresh Air to the individual floor mounted AHUs intake there the intake is controlled is controlled by Modulating Damper.
 Sensor is located in 2/3 of duct from the supply plenum.

Reply:
Since this fan is with the supply air duct changing the type from spc to sspc.